activerecord/test/cases/normalized_attribute_test.rb

```diff
@@ -7,7 +7,7 @@
 class NormalizedAttributeTest < ActiveRecord::TestCase
   class NormalizedAircraft < Aircraft
     normalizes :name, with: -> name { name.titlecase }
-    normalizes :manufactured_at, with: -> time { time.noon }
+    normalizes [:manufactured_at], with: -> time { time.noon }
```

Comment:
Curious why the array is necessary here?

Reply:
It is to exercise [`names.flatten!`](https://github.com/rails/rails/pull/49519/files#diff-c8f1230590d17006bf5f5e325481d99e59c847378a565636ab8133cfb3c0d455R81).  That way, if someone accidentally removes `names.flatten!`, the tests will fail.

Reply:
Nice! Maybe we can add an example like `normalizes [:name, :hometown], with: -> value { value.downcase }` somewhere? This wasn't supported before, right?